src/components/fields/text-field/text-field.tsx

```diff
@@ -98,6 +98,11 @@ export const TextField = (props: IGenericFieldProps<ITextFieldSchema | IEmailFie
 		if (uiType === "numeric-field" && !event.target.value) {
 			onChange({ target: { value: undefined } });
 		} else {
+			const inputValue = event.target.value;
+			event.target.value =
+				customOptions?.uppercase && inputValue !== inputValue.toUpperCase()
```

Comment:
why the need to compare against an uppercased version instead of outrightly converting to uppercase?

since numeric field doesn't support uppercase, there is a need to ensure its custom options do not accept that

Reply:
we ensure only text-fields run the uppercase only through a check
and also typing the custom options

Reply:
removed uppercase check. 

was needed on bsg side because FEE form onchange triggers on updating control values, resulting in infinite loop.